workshop/content/exercises/D-01-content.md

```diff
@@ -107,16 +112,37 @@ Run this command and click on the link in terminal 1
 echo https://github.com/$GITHUB_ORG/cat-service-release
 ```
 
+### If not public make the `cat-service-release` repo public
+For Github Actions to work the `cat-service-release` repo must be public. If you need to make the repository public. Within the repo `cat-service-release` in your Github org, navigate to `Settings`.
```

Comment:
Your 2nd sentence is repetitive. Remove.

*GitHub

Reply:
If they create it rather than clone, they can make it public from the outset.

Reply:
deleted section